includes/Runtime.php

```diff
@@ -39,6 +39,8 @@ public function loadIntoPage( $page ) {
 	public function prepareRuntime() {
 		global $wp_version;
 		$sdk = apply_filters( 'newfold-runtime', array( 'wpversion' => $wp_version ) );
+		
+		$sdkProps = apply_filters( 'newfold_runtime', array( 'wpversion' => $wp_version ) );
```

Comment:
I think we should just do a `return apply_filters( 'newfold_runtime', array( <all the things> ) );` and not worry about any "default" values. We're the only ones using this, and this would add more clarity and prevent future bugs. As it stands, the current code adds multiple `wpversion` properties to the array.